docs/config-file-ort-yml.md

```diff
@@ -235,3 +235,33 @@ resolutions:
     reason: "LICENSE_ACQUIRED_EXCEPTION"
     comment: "Commercial Qt license for the project was purchased, for details see https://jira.example.com/issues/SOURCING-5678"
 ```
+
+### Resolving Vulnerabilities
```

Comment:
The [Resolution Basics section](https://github.com/oss-review-toolkit/ort/blob/master/docs/config-file-ort-yml.md#resolution-basics) should be updated: Resolutions allow you to resolve issues, policy rule violations **or vulnerabilities** by marking them as acceptable. A resolution is applied to specific issues or violations via the regular expression specified in the message of a resolution.